app/scripts/controllers/common.js

```diff
@@ -35,6 +35,15 @@ var commons = angular.module('common', ['stats', 'udc'])
       $scope.load15 = data.load[2] == '-.-' ? data.load[2] : data.load[2].toFixed(2);
       $scope.version = data.version;
       $scope.docs_url = getDocsUrl(data.version);
+      $scope.checks = data.checks;
+      $scope.config_label = "success";
+      if (data.checks.length > 0) {
+        data.checks.filter(function(obj) {
+          $scope.config_label = (obj.severity === 1) ? "info" : $scope.config_label;
```

Comment:
defined the full label name here, e.g. `label-info` instead of doing string concat in the template